kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunnerViewModel.kt

```diff
@@ -38,14 +39,15 @@ internal class WorkflowRunnerViewModel<OutputT : Any>(
     private val workflow: Workflow<InputT, OutputT, Any>,
     private val viewRegistry: ViewRegistry,
     private val inputs: Flowable<InputT>,
-    savedInstanceState: Bundle?
+    savedInstanceState: Bundle?,
+    private val dispatcher: CoroutineDispatcher = Dispatchers.Main.immediate
```

Comment:
This parameter should be a `CoroutineContext` so it can play nice with scoped concurrency, coroutine names for troubleshooting, etc.

Reply:
Oh, flatMapWorkflow is dumb. Need to delete…